codegen/lib/graphql_swift_gen/templates/type.swift.erb

```diff
@@ -19,7 +19,9 @@ import Foundation
 
 extension <%= schema_name %> {
   <% case type.kind; when 'OBJECT', 'INTERFACE', 'UNION' %>
-    open class <%= type.name %>Query: GraphQL.AbstractQuery {
+    open class <%= type.name %>Query: GraphQL.AbstractQuery, GraphQLQuery {
+      public typealias Response = <%= type.name %>
```

Comment:
Note that if the response is an interface, that Response will point to the protocol defined above, not to the Unknown class below